pkg/webhook/parser_integration_test.go

```diff
@@ -80,6 +82,23 @@ var _ = Describe("Webhook Generation From Parsing to CustomResourceDefinition",
 		}
 		assertSame(actualMutating, expectedMutating)
 		assertSame(actualValidating, expectedValidating)
+
+		By("loading the generated v1beta1 YAML")
+		actualFile, err = ioutil.ReadFile(path.Join(outputDir, "manifests.v1beta1.yaml"))
+		Expect(err).NotTo(HaveOccurred())
+		actualMutating, actualValidating = unmarshalBoth(actualFile)
+
+		By("loading the desired v1beta1 YAML")
+		expectedFile, err = ioutil.ReadFile("manifests.v1beta1.yaml")
+		Expect(err).NotTo(HaveOccurred())
+		expectedMutating, expectedValidating = unmarshalBoth(expectedFile)
+
+		By("comparing the two")
+		assertSame = func(actual, expected interface{}) {
```

Comment:
`assertSame` is already defined above. Perhaps define this function globally.

Reply:
Fixed